docs: disambiguation simplification for non-react solutions (#2054)
microsoftgraph/microsoft-graph-toolkit: packages/mgt-spfx-utils/README.md

By disambiguating tag names of Microsoft Graph Toolkit components, you can use your own version of MGT rather than using the centrally deployed `@microsoft/mgt-spfx` package. This allows you to avoid colliding with SharePoint Framework components built by other developers. When disambiguating tag names, MGT is included in the generated SPFx bundle, increasing its size. It is strongly recommended that you use a disambiguation value unique to your organization and solution to avoid collisions with other solutions, e.g. `contoso-hr-extensions`.

> **Important:** Since a given web component tag can only be registered once this approach **must** be used along with the `customElementHelper.withDisambiguation('foo')` as this allows developers to create disambiguated tag names.

### When using no framework web parts

> When building SharePoint Framework web parts without a JavaScript framework the `@microsoft/mgt-components` library must be asynchronously loaded after configuring the disambiguation setting. No helper library is necessary as dynamic imports are sufficient. After the browser has loaded the `@microsoft/mgt-components` library any tags which may have already been rendered to the DOM are automatically upgraded and have the custom behavior attached to them.

Reviewer: Are they upgraded to the latest version? If that's the case maybe we should be explicit here.
```suggestion
When building SharePoint Framework web parts without a JavaScript framework the `@microsoft/mgt-components` library must be asynchronously loaded after configuring the disambiguation setting. No helper library is necessary as dynamic imports are sufficient. After the browser has loaded the `@microsoft/mgt-components` library any tags which may have already been rendered to the DOM are automatically upgraded to the latest version and have the custom behavior attached to them.
```

Author: The DOM elements here are upgraded in the sense that they have the behavior of the custom element attached to them.

Because there can be only one class for a given tag name using "latest version" here is misleading and might have folks assume that a tag registered with mgt-login using v2.x might get "upgraded" to use the logic being attached to mgt-contoso-login using 3.x.

I'll reword this to be a bit more succinct and remove the upgrade wording. Thanks for catching this.

> When building SharePoint Framework web parts without a JavaScript framework the `@microsoft/mgt-components` library must be asynchronously loaded after configuring the disambiguation setting. No helper library is necessary as dynamic imports are sufficient. After the browser has loaded the `@microsoft/mgt-components` library any tags which may have already been rendered to the DOM are automatically upgraded and have the custom behavior attached to them.

Author: ```suggestion
When building SharePoint Framework web parts without a JavaScript framework the `@microsoft/mgt-components` library must be asynchronously loaded after configuring the disambiguation setting. No helper library is necessary as dynamic imports are sufficient. After the browser has loaded the `@microsoft/mgt-components` library any tags which had already been rendered to the DOM without an existing custom element registration will automatically have the loaded custom behavior attached to them.
```
